docs/api/resource.rst

```diff
@@ -1,28 +1,275 @@
+.. _resource-endpoints:
+
 ##################
 Resource endpoints
 ##################
 
-.. _resource-endpoints:
+GET /buckets/syncto/collections/{collection}/records
```

Comment:
You are right, [I like it](http://payments.readthedocs.org/en/latest/design/api/storage.html#storage-api)